controllers/managedocs_controller.go

```diff
@@ -102,6 +108,7 @@ const (
 	convergedDeploymentType                = "converged"
 	consumerDeploymentType                 = "consumer"
 	providerDeploymentType                 = "provider"
+	caBundleConfigMapName                  = "cert-ca-bundle"
```

Comment:
- a different name would be good to state prometheus is using this bundle
- not a hard req, good to have IMHO

Reply:
when you say `name` did you mean the variable name `caBundleConfigMapName` or the value part `"cert-ca-bundle"`?

Reply:
value.

Reply:
Changed